### PR TITLE
77 update profile settings UI

### PR DESCRIPTION
**Pull request by mabelene**
image
image

### CODE REVIEW

File: components/Dashboard/Modals/ProfileDialog/index.js
Comment (LemonFace0309) [inline components/Dashboard/Modals/ProfileDialog/index.js:12]: please keep imports consistent. We shouldn't have spaces between imports from external packages. Reference the other pages to see what I mean.

File: components/Dashboard/Modals/ProfileDialog/index.js
Comment (LemonFace0309) [inline components/Dashboard/Modals/ProfileDialog/index.js:156]: beautiful, thanks for using `classNames`

File: pages/dashboard/index.js
Comment (LemonFace0309) [inline pages/dashboard/index.js:10]: please remove space in line 8. Also try to group mui (material ui) imports together instead of having `classnames` separating them.

**Review by LemonFace0309 — APPROVED** [review]
LGTM, please address and resolve issues before merging. Also please look at your console in google dev tools. You're getting a ton of errors there that you're not addressing. This really should not go into production. I will fix it for you this time but please do that yourself next time.

Example:
image
image

**Comment by LemonFace0309** [comment]
image

please make this response too

**Comment by LemonFace0309** [comment]
image

lmao sometimes if you're wondering why things aren't working, console errors should give you a hint.

**Comment by LemonFace0309** [comment]
image

looks like you also introduced a horizontal sidebar. This is likely due to the fact that you made one of your grid components a container which takes extra padding. I will fix for you but should be aware of these issues

**Comment by mabelene** [comment]
changes fixed, will merge if 
image
is not an issue

**Review by jimmyygit — APPROVED** [review]
if ur not using backend, socket errors are usually not an issue 👍

**Pull request merged**
